test/test_business_days.rb

```diff
@@ -90,6 +90,14 @@
       assert_equal expected, later
     end
 
+    it "should ignore business hour when adding one business day from after business hours" do
+      BusinessTime::Config.consider_business_hours_for_rolls = false
+      wednesday = Time.parse("Wednesday October 14th, 2015, 21:54 pm")
+      later = 1.business_days.after(wednesday)
+      expected = Time.parse("Friday October 15th, 2015, 21:54 pm")
```

Comment:
Shouldn't it actually be at the start of next workday? It seems strange that a roll that considers business hours returns a time that's outside the business hours.
```ruby
expected = Time.parse("Friday October 15th, 2015, #{BusinessTime.config.beginning_of_workday}")
```